### PR TITLE
[ContainerInstance] Address self-review comments

### PR DESCRIPTION
**Pull request by fengzhou-msft**
# Contributing to the Azure SDK

Please see our [CONTRIBUTING.md](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md) if you are not familiar with contributing to this repository or have questions.

For specific information about pull request etiquette and best practices, see [this section](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md#pull-request-etiquette-and-best-practices).

### CODE REVIEW

File: sdk/containerinstance/Azure.ResourceManager.ContainerInstance/src/autorest.md [hunk 1]
```diff
@@ -44,36 +43,41 @@ rename-rules:
...
-      $.ContainerExecResponse["x-ms-client-name"] = "ContainerExecResult";
-      $.Capabilities["x-ms-client-name"] = "ContainerInstanceCapabilities";
-      $.ContainerGroupSubnetId.properties.id["x-ms-format"] = "arm-id";
-      $.InitContainerDefinition["x-ms-client-name"] = "InitContainerDefinitionContent";
-      $.LogAnalytics.properties.workspaceId["x-ms-format"] = "arm-id";
-  - from: swagger-document
-    where: $.definitions.ContainerHttpGet.scheme["x-ms-enum"]
-    transform: $["name"] = "ContainerInstanceScheme"
+rename-mapping:
+  Logs: ContainerLogs
```
Comment (ArthurMa1978) [inline sdk/containerinstance/Azure.ResourceManager.ContainerInstance/src/autorest.md:63]: Is `ConstainerLogs` still name collision with other RPs?

File: sdk/containerinstance/Azure.ResourceManager.ContainerInstance/api/Azure.ResourceManager.ContainerInstance.netstandard2.0.cs [hunk 2]
```diff
@@ -553,8 +553,8 @@ public partial class UsageName
     public partial class VolumeMount
```
Comment (ArthurMa1978) [inline sdk/containerinstance/Azure.ResourceManager.ContainerInstance/api/Azure.ResourceManager.ContainerInstance.netstandard2.0.cs:553]: Is `VolumeMount` still colliding with other RPs?

File: sdk/containerinstance/Azure.ResourceManager.ContainerInstance/src/autorest.md [hunk 3]
```diff
@@ -44,36 +43,41 @@ rename-rules:
...
+  Event: ContainerEvent
```
Comment (ArthurMa1978) [inline sdk/containerinstance/Azure.ResourceManager.ContainerInstance/src/autorest.md:64]: Is `ConstainerEvent` still colliding with other RPs?

**Review by ArthurMa1978 — APPROVED** [review]

File: sdk/containerinstance/Azure.ResourceManager.ContainerInstance/src/autorest.md (hunk 3 above)
Comment (fengzhou-msft) [reply sdk/containerinstance/Azure.ResourceManager.ContainerInstance/src/autorest.md:65]: No

File: sdk/containerinstance/Azure.ResourceManager.ContainerInstance/api/Azure.ResourceManager.ContainerInstance.netstandard2.0.cs (hunk 2 above)
Comment (fengzhou-msft) [reply sdk/containerinstance/Azure.ResourceManager.ContainerInstance/api/Azure.ResourceManager.ContainerInstance.netstandard2.0.cs]: Updated

File: sdk/containerinstance/Azure.ResourceManager.ContainerInstance/src/autorest.md (hunk 1 above)
Comment (fengzhou-msft) [reply sdk/containerinstance/Azure.ResourceManager.ContainerInstance/src/autorest.md]: Updated

**Pull request merged**